wagtail_localize/models.py

```diff
@@ -363,10 +363,10 @@ def export_po(self):
         Exports a PO file contining the source strings.
         """
         # Get messages
-        messages = defaultdict(list)
+        messages = set()
```

Comment:
Keep in mind a `set()` won't maintain its ordering.

Reply:
Oh, I assumed they worked the same way as a dict! thanks for spotting, we definitely want that ordered

Reply:
Fixed, I just turned it into a list. There isn't really a need to deduplicate things anyway!